core/crypto/vrf/p256/p256_test.go

```diff
@@ -66,6 +74,60 @@ func TestH2(t *testing.T) {
 	}
 }
 
+func TestFromWrappedKey(t *testing.T) {
+	ctx := context.Background()
+	for _, tc := range []struct {
+		spec   *keyspb.Specification
+		keygen keys.ProtoGenerator
+	}{
+		{
+			spec: &keyspb.Specification{
+				Params: &keyspb.Specification_EcdsaParams{
+					EcdsaParams: &keyspb.Specification_ECDSA{
+						Curve: keyspb.Specification_ECDSA_P256,
+					},
+				},
+			},
+			keygen: func(ctx context.Context, spec *keyspb.Specification) (proto.Message, error) {
+				return der.NewProtoFromSpec(spec)
+			},
+		},
+	} {
+		// Generate VRF key.
+		wrapped, err := tc.keygen(ctx, tc.spec)
+		if err != nil {
+			t.Errorf("NewProtoFromSpec failed: %v", err)
```

Comment:
There's no guarantee that `tc.keygen` will call `NewProtoFromSpec`, so this error message could be misleading in the future.

Reply:
If you're going to leave this as a tabular test, either all of the error messages should be prefixed with `tc.desc` or the body of the test should be run in a subtest (I recommend the latter).

Reply:
done